src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslatorHelper.cs

```diff
@@ -0,0 +1,1781 @@
+				{ "Laoo", new char[] { '\u0ED0', '\u0ED1', '\u0ED2', '\u0ED3', '\u0ED4', '\u0ED5', '\u0ED6', '\u0ED7', '\u0ED8', '\u0ED9' } },
+				{ "Latn", new char[] { '\u0030', '\u0031', '\u0032', '\u0033', '\u0034', '\u0035', '\u0036', '\u0037', '\u0038', '\u0039' } },
+				{ "Lepc", new char[] { '\u1C40', '\u1C41', '\u1C42', '\u1C43', '\u1C44', '\u1C45', '\u1C46', '\u1C47', '\u1C48', '\u1C49' } },
+				{ "Limb", new char[] { '\u1946', '\u1947', '\u1948', '\u1949', '\u194A', '\u194B', '\u194C', '\u194D', '\u194E', '\u194F' } },
+				{ "Mlym", new char[] { '\u0D66', '\u0D67', '\u0D68', '\u0D69', '\u0D6A', '\u0D6B', '\u0D6C', '\u0D6D', '\u0D6E', '\u0D6F' } },
+				{ "Mong", new char[] { '\u1810', '\u1811', '\u1812', '\u1813', '\u1814', '\u1815', '\u1816', '\u1817', '\u1818', '\u1819' } },
+				{ "Mtei", new char[] { '\uABF0', '\uABF1', '\uABF2', '\uABF3', '\uABF4', '\uABF5', '\uABF6', '\uABF7', '\uABF8', '\uABF9' } },
+				{ "Mymr", new char[] { '\u1040', '\u1041', '\u1042', '\u1043', '\u1044', '\u1045', '\u1046', '\u1047', '\u1048', '\u1049' } },
+				{ "MymrShan", new char[] { '\u1090', '\u1091', '\u1092', '\u1093', '\u1094', '\u1095', '\u1096', '\u1097', '\u1098', '\u1099' } },
+				{ "Nkoo", new char[] { '\u07C0', '\u07C1', '\u07C2', '\u07C3', '\u07C4', '\u07C5', '\u07C6', '\u07C7', '\u07C8', '\u07C9' } },
+				{ "Olck", new char[] { '\u1C50', '\u1C51', '\u1C52', '\u1C53', '\u1C54', '\u1C55', '\u1C56', '\u1C57', '\u1C58', '\u1C59' } },
+				{ "Orya", new char[] { '\u0B66', '\u0B67', '\u0B68', '\u0B69', '\u0B6A', '\u0B6B', '\u0B6C', '\u0B6D', '\u0B6E', '\u0B6F' } },
+				{ "Saur", new char[] { '\uA8D0', '\uA8D1', '\uA8D2', '\uA8D3', '\uA8D4', '\uA8D5', '\uA8D6', '\uA8D7', '\uA8D8', '\uA8D9' } },
+				{ "Sund", new char[] { '\u1BB0', '\u1BB1', '\u1BB2', '\u1BB3', '\u1BB4', '\u1BB5', '\u1BB6', '\u1BB7', '\u1BB8', '\u1BB9' } },
+				{ "Talu", new char[] { '\u19D0', '\u19D1', '\u19D2', '\u19D3', '\u19D4', '\u19D5', '\u19D6', '\u19D7', '\u19D8', '\u19D9' } },
+				{ "TamlDec", new char[] { '\u0BE6', '\u0BE7', '\u0BE8', '\u0BE9', '\u0BEA', '\u0BEB', '\u0BEC', '\u0BED', '\u0BEE', '\u0BEF' } },
+				{ "Telu", new char[] { '\u0C66', '\u0C67', '\u0C68', '\u0C69', '\u0C6A', '\u0C6B', '\u0C6C', '\u0C6D', '\u0C6E', '\u0C6F' } },
+				{ "Thai", new char[] { '\u0E50', '\u0E51', '\u0E52', '\u0E53', '\u0E54', '\u0E55', '\u0E56', '\u0E57', '\u0E58', '\u0E59' } },
+				{ "Tibt", new char[] { '\u0F20', '\u0F21', '\u0F22', '\u0F23', '\u0F24', '\u0F25', '\u0F26', '\u0F27', '\u0F28', '\u0F29' } },
+				{ "Vaii", new char[] { '\uA620', '\uA621', '\uA622', '\uA623', '\uA624', '\uA625', '\uA626', '\uA627', '\uA628', '\uA629' } },
+			};
+
+			Debug.Assert(result.Count == Count);
+			return result;
+		}
+
+		public static string GetNumeralSystem(string languageTag)
+		{
+			languageTag = languageTag.ToLowerInvariant();
+
+			if (_languageTagToNumeralSystemDictionary.TryGetValue(languageTag, out string value))
```

Comment:
With regards to performance, it's best to use an actual switch/case block. The C# compiler uses a static hash-table that's more efficient at runtime and particularly initialization time.